canonical/rebac-admin
# WD-13045 - refactor: use ids for operating on entities

## Done

- Use entity ids for all operations.
- Temporarily implement selecting identites and roles from the API.

## QA

- Go to a form such as for adding a group.
- Fill in the form and add some related entities and submit.
- The group should get added and opening the edit form should show the related entities.

## Details

https://warthogs.atlassian.net/browse/WD-13045
